src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -7070,7 +7069,7 @@ private BoundExpression BindArrayAccess(ExpressionSyntax node, BoundExpression e
             TypeSymbol resultType = rank == 1 &&
                 TypeSymbol.Equals(
                     convertedArguments[0].Type,
-                    Compilation.GetWellKnownType(WellKnownType.System_Range),
+                    Compilation.GetWellKnownType(WellKnownType.System_Range, recordUsage: false),
```

Comment:
>recordUsage: false [](start = 77, length = 18)

Similar question: wouldn't we want to record usage here unless there are errors?

Reply:
> Similar question: wouldn't we want to record usage here unless there are errors?

I don't think we should say that we use System.Range here if the ```convertedArguments[0].Type``` is not it. Do we report any error here if System.Range is missing? We use ```convertedArguments[0].Type``` and that will be tracked elsewhere by other means.

---
In reply to: [351357811](https://github.com/dotnet/roslyn/pull/40010#discussion_r351357811) [](ancestors = 351357811)